scripts/helpers/eosio.sh

```diff
@@ -300,72 +300,48 @@ function build-clang() {
     fi
 }
 
-function install_libpqxx_from_source() {
-    if [ ! -d ${OPT_DIR}/pqxx ]; then
-        execute bash -c "cd $SRC_DIR && \
-            curl -L https://github.com/jtv/libpqxx/archive/7.2.1.tar.gz | tar zxvf - && \
-            cd libpqxx-7.2.1 && mkdir build && cd build && \
-            ${CMAKE} $CXX_SPEC $EXTRA_CMAKE_FLAGS -DCMAKE_INSTALL_PREFIX=${OPT_DIR}/pqxx -DSKIP_BUILD_TEST=ON -DCMAKE_BUILD_TYPE=Release .. && \
-            make -j${JOBS} && make install && \
-            cd ../.. && rm -rf libpqxx-7.2.1"
-    fi
-
-    if [ -z "$PKG_CONFIG_PATH" ]; then
-        export PKG_CONFIG_PATH="${OPT_DIR}/pqxx/lib/pkgconfig"
-    else
-        export PKG_CONFIG_PATH="${PKG_CONFIG_PATH}:${OPT_DIR}/pqxx/lib/pkgconfig"
-    fi
-} 
-
-function ensure-libpq-and-libpqxx() {  
-
+function ensure-libpq() {
     if [[ $ARCH == "Darwin" ]]; then
-        #install libpq
-        [ -f /usr/local/opt/libpq/lib/pkgconfig/libpq.pc ] || brew install libpq
-        #install libpqxx
-        if $PIN_COMPILER; then
-            EXTRA_CMAKE_FLAGS="-DPostgreSQL_ROOT=/usr/local/opt/libpq"
-            install_libpqxx_from_source
-        elif [ ! -f /usr/local/lib/pkgconfig/libpqxx.pc ]; then
-            brew install libpqxx
-        fi
-    fi
+        #build libpq from source
+        curl -L https://github.com/postgres/postgres/archive/refs/tags/REL_13_3.tar.gz | tar zxvf -
+        cd postgres-REL_13_3
+        ./configure && make && sudo make install && cd .. && rm -rf postgres-REL_13_3
 
+        sudo rm -rf /usr/local/var/postgres
```

Comment:
This looks really dangerous. Could it eliminate a user's database?

Reply:
imo for scripted builds doesn't seem worth trying to do anything with the database. Running the postgres tests outside of CI seems to require either
* user able to run docker, or
* manually setting PGHOST
So it's doesn't seem reasonable for this to just work as there is no way the script can safely make it just work.

Reply:
The reason we remove old path of the test data is to give new test a clean space.  
Remove this line, the first time install eos will OK. but if user install eos and uninstall it and then reinstall eos, there would be some unexpected table in db.  Can we take the eos host as an dedicated machine?

Reply:
OK, removed. so no issue now.